ADD  -- Task 6(I2): Add entity activities

marvicmar opened the pull request.

marvicmar committed:
Add entity Activites

marvicmar committed:
Update entity activites

alepervaz reviewed (approved):
el codigo se encuentra completo y de forma correcta

alesanfe reviewed (changes requested):
Creo que debería de ser TIMESTAMP.

marvicmar committed:
Update Activites.java

Change DATE from TIMESTAMP

alesanfe reviewed (changes requested):
Creo que si acaso faltan los comentarios para indicar cada una de las partes de la entidad.

alesanfe committed:
fix(init): Add entity activities

Añadidos los comentarios para que la entidad siga el formato de las entidades del resto del trabajo.

alepervaz reviewed (approved):
todo correcto
